src/compile/unit.ts

```diff
@@ -234,14 +234,13 @@ export class UnitModel extends Model {
 
         // We no longer support false in the schema, but we keep false here for backward compatability.
         if (axisSpec !== null && axisSpec !== false) {
-          let vlAxisBase: VlAxisBase = {};
-          AXIS_BASE_PROPERTIES.forEach(function(property) {
-            if (encoding !== undefined && encoding[property] !== undefined) {
-              vlAxisBase[property] = encoding[property];
+          let vlAxisBase: VlOnlyAxisConfig = {};
```

Comment:
We no longer have VlAxisBase types, so you should rename all relevant variable names.